ts/Extensions/Annotations/NavigationBindings.ts

```diff
@@ -1257,8 +1257,14 @@ setOptions({
          * - `end`: last event to be called after last step event
          *
          * @type         {Highcharts.Dictionary<Highcharts.NavigationBindingsOptionsObject>|*}
+         *
          * @sample       stock/stocktools/stocktools-thresholds
          *               Custom bindings in Highcharts Stock
+         * @sample       highcharts/annotations/bindings/
+         *               Simple bingind without Stock Tools
+         * @sample       highcharts/annotations/bindings-custom-annotation/
+         *               Custom annotation binding without Stock Tools
```

Comment:
1. I think there's no need to mention it it without Stock Tools. Simply add `@sample {highcharts} highcharts/...` so it will be available only under Highcharts API 
2. Could you also add the product to the first demo? So the Stock demo will be available only in Highcharts Stock API. Like this:
```
         * @sample {highstock} stock/stocktools/stocktools-thresholds
         *               Custom bindings in Highcharts Stock
```